lib/routes/oauth.js

```diff
@@ -0,0 +1,187 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var validators = require('./validators')
+var HEX_STRING = validators.HEX_STRING
+var BASE64_JWT = validators.BASE64_JWT
+
+var butil = require('../crypto/butil')
+
+var URL = require('url')
+var Request = require('request')
+var jwtool = require('fxa-jwtool')
```

Comment:
It seems like the auth-server way is to pass dependencies as arguments to `module.exports`, but sometimes not. I don't know what the rules are for that, but just pointing at places for things.